packages/sdk/src/sdkBase.ts

```diff
@@ -999,7 +998,7 @@ export class NxtpSdkBase {
     fulfillSignature: string,
     decryptedCallData: string,
     relayerFee = "0",
-    useRelayers = true,
+    _useRelayers = false,
```

Comment:
how hot is the hot fix? i dont want to change the default behavior here without hearing from @rhlsthrm

Reply:
more of a moderate warmfix imo

Reply:
yes this is 100% for @rhlsthrm to pick up from here